Lib/xml/etree/ElementTree.py

```diff
@@ -622,11 +621,10 @@ def iter(self, tag=None):
 
     # compatibility
     def getiterator(self, tag=None):
-        # Change for a DeprecationWarning in 1.4
         warnings.warn(
             "This method will be removed in future versions.  "
             "Use 'tree.iter()' or 'list(tree.iter())' instead.",
-            PendingDeprecationWarning, stacklevel=2
+            DeprecationWarning, stacklevel=2
```

Comment:
Do you want to update the messages to have the version since the deprecation and the planned removal version here (and on line 418) ?

Reply:
It is in the documentation.